spec/generate_pairs_spec.rb

```diff
@@ -46,12 +46,13 @@ module MakersToolbelt
     describe 'result' do
       it 'contains a JSON formatted one-factorization of the names in the file' do
         names = GeneratePairs.load_names(READ_PATH)
+
+        allow(GeneratePairs).to receive(:load_names).and_return(double(one_factorize: double(shuffle: names.one_factorize)))
```

Comment:
in general we should avoid stubbing methods on the object under test no?  As @pitchinvasion has suggested before, might it be better here to extract the dependency so that a double can be passed in.  Although perhaps that's overkill when it comes to interacting with the filesystem ...?